README.md

```diff
@@ -1,2 +1,11 @@
-# agent-modules
-A repository of public agent modules
+<p align="center"><img src="assets/logo_and_name.png" alt="Grafana Agent logo"></p>
+
+[Modules](https://grafana.com/docs/agent/latest/flow/concepts/modules/) are a 
+way to create Grafana Agent [Flow](https://grafana.com/docs/agent/latest/flow/)
+configurations which can be loaded as a component. Modules are a great way to
+parameterize a configuration to create reusable pipelines.
+
+## Contents
+- <b>modules</b>: A library of usable modules out of the box
+- <b>example</b>: A practical example shown for each module loader plus without modules for comparison
+- <b>util</b>: Utilities for managing modules in this repo
```

Comment:
Also thoughts on having a table listing all the modules by category header with a table listing
* Name
* Link
* Short description